packages/neuron-wallet/src/services/networks.ts

```diff
@@ -66,6 +66,7 @@ enum NetworksKey {
 }
 
 const oldDefaultNames = ['Default', 'default node', presetNetworks.networks[0].name]
+const oldDefaultRemotes = ['http://localhost:8114', BUNDLED_CKB_URL]
```

Comment:
`http://localhost:8114` is magical, maybe it can be defined in `constants` with why `localhost` is deprecated.

Reply:
`oldDefaultRemotes` is a const variable, I guess adding a comment above the `oldDefaultRemotes` is enough.

Reply:
LGTM